frontend/src/hooks/useApi.ts

```diff
@@ -0,0 +1,26 @@
+import CustomAxios from 'customAxios';
+import { useDispatch } from 'react-redux';
+import { showLoading, hideLoading } from 'react-redux-loading-bar';
+
+export const useApi = () => {
```

Comment:
Thank you for doing this! I've been meaning to add this for a while.
Love it!

Reply:
I am sad that I did not have the use of it in my table. But yeah, it centralizes the loading control